### PR TITLE
Verification of PivotalTracker

### PR DESCRIPTION
**Pull request by Gee19**
* Add missing scope to PivotalFields
* Remove Alm context section
* Remove base alm_project help_text/description
* Update pt_group_label to pt_issue_label, keep old name.

Refs #SDE-3970

### CODE REVIEW

File: sdetools/modules/sync_pt/pt_metadata.py [hunk 1]
```diff
@@ -34,26 +34,29 @@ def __init__(self, phases):
...
             field_type=FieldType.STRING,
-            placeholder='accepted',
-            default='accepted'
+            default='accepted',
+            scope=['system']
         )
         self.pt_new_status = IntegrationField(
             name='pt_new_status',
-            label=_('Open a story in %s with the following state' % self.name),
+            label=_('Open a task in %s with the following state' % self.name),
```
Comment (lmontopo) [inline sdetools/modules/sync_pt/pt_metadata.py]: Maybe it actually makes more sense for it to be story?  That keeps it consistent with the "Story Type" field?  Unless there are both stories and tasks, and they mean different things in PT?

**Comment by lmontopo** [comment]
Had one comment, otherwise it looks great!

File: sdetools/alm_integration/alm_metadata_plugin_base.py [hunk 2]
```diff
@@ -400,7 +400,7 @@ def __init__(self, phases):
             max_length=255,
             required=True,
             scope=['project'],
-            help_text=_("Project name in %s" % self.name)
+            description=_("Project name in %s" % self.name)
```
Comment (sibartlett) [inline sdetools/alm_integration/alm_metadata_plugin_base.py]: It might be best to drop description completely here. It's not always the project's name.

Comment (lmontopo) [inline sdetools/alm_integration/alm_metadata_plugin_base.py]: I agree with that too, actually.  I can make this change quick.

File: sdetools/modules/sync_pt/pt_metadata.py [hunk 3]
```diff
@@ -34,26 +34,29 @@ def __init__(self, phases):
...
-            placeholder='unscheduled',
             default='unscheduled',
+            scope=['system']
         )
-        self.pt_group_label = IntegrationField(
-            name='pt_group_label',
-            label=_('Label to assign to issues'),
-            field_type=FieldType.STRING
+        self.pt_issue_label = IntegrationField(
+            name='pt_issue_label',
```
Comment (sibartlett) [inline sdetools/modules/sync_pt/pt_metadata.py]: Then name should be `pt_group_label`. We can't change names of params that already exist.

Comment (lmontopo) [inline sdetools/modules/sync_pt/pt_metadata.py]: ahh good catch.

File: sdetools/modules/sync_pt/pt_metadata.py (hunk 1 above)
Comment (Gee19) [inline sdetools/modules/sync_pt/pt_metadata.py]: I just googled the terminology used by Pivotal and noticed they mentioned tasks, either works though

File: sdetools/modules/sync_pt/pt_metadata.py (hunk 3 above)
Comment (Gee19) [inline sdetools/modules/sync_pt/pt_metadata.py]: Fixed

File: sdetools/alm_integration/alm_metadata_plugin_base.py (hunk 2 above)
Comment (Gee19) [inline sdetools/alm_integration/alm_metadata_plugin_base.py]: Fixed

File: sdetools/modules/sync_pt/pt_metadata.py (hunk 1 above)
Comment (lmontopo) [inline sdetools/modules/sync_pt/pt_metadata.py]: yea maybe its not a big deal, not sure.

File: sdetools/modules/sync_pt/pt_metadata.py [hunk 4]
```diff
@@ -34,26 +34,29 @@ def __init__(self, phases):
...
             name='pt_new_status',
-            label=_('Open a story in %s with the following state' % self.name),
+            label=_('Open a task in %s with the following state' % self.name),
             field_type=FieldType.STRING,
-            placeholder='unscheduled',
             default='unscheduled',
+            scope=['system']
         )
-        self.pt_group_label = IntegrationField(
+        self.pt_issue_label = IntegrationField(
```
Comment (sibartlett) [inline sdetools/modules/sync_pt/pt_metadata.py]: Probably should keep this pt_group_label too, to be consistent.

Comment (Gee19) [inline sdetools/modules/sync_pt/pt_metadata.py]: Fixed

**Pull request merged**